pkg/ctl/create/cluster.go

```diff
@@ -253,6 +253,12 @@ func doCreateCluster(p *api.ProviderConfig, cfg *api.ClusterConfig, nameArg stri
 		}
 		logger.Info("nodegroup %q will use %q [%s/%s]", ng.Name, ng.AMI, ng.AMIFamily, cfg.Metadata.Version)
 
+		// Lookup AMI root device name
+		if err := ctl.GetRootDevice(ng); err != nil {
```

Comment:
Let's just roll it up into `ctl.EnsureAMI`.

Reply:
Will do!